docs/components-upgrade.md

```diff
@@ -67,20 +61,20 @@ acs:
 Once these changes are saved run the command below:
 
 ```bash
-ansible-playbook playbooks/acs.yml -i inventory_ssh.yml -e "@7.0.N-extra-vars.yml"
+ansible-playbook playbooks/acs.yml -i inventory_ssh.yml -e "acs_play_major_version=23"
 ```
 
 > Note: Use whatever inventory and config file that matches your use case
-> If you're applying a hotfix to the latest major release (7.1 as of writing) you don't need to specify an extra config file with "-e @file"
+> If you're applying a hotfix to the latest major release (23 as of writing) you don't need to specify an extra config file with "-e acs_play_major_version=xx"
```

Comment:
Then make sure your example above do not use 23. Or comment the other way around